rules/windows/defense_evasion_msbuild_making_network_connections.toml

```diff
@@ -116,11 +121,28 @@ tags = [
 type = "eql"
 
 query = '''
-sequence by process.entity_id
-  [process where host.os.type == "windows" and process.name : "MSBuild.exe" and event.type == "start"]
+sequence by process.entity_id with maxspan=30s
+
+  /* Look for MSBuild.exe process execution */
+  /* The events for this first sequence may be noisy, consider adding exceptions */
+  [process where host.os.type == "windows"
+    and (
+          process.pe.original_file_name: "MSBuild.exe" or
+          process.name: "MSBuild.exe"
+        )
+    and event.type == "start" and user.id != "S-1-5-18"]
+
+  /* Followed by a network connection to an external address */
+  /* Exclude domains that are known to be benign */
   [network where host.os.type == "windows" and process.name : "MSBuild.exe" and
```

Comment:
>process.name : "MSBuild.exe" and

May cause FNs as we are also looking for `process.pe.original_file_name: "MSBuild.exe" ` in the first event 🤔

Reply:
@w0rk3r - Would it make sense to include original file name in the sequenced network event as well then or just look for original file name?

Reply:
We had some issues in populating the original file name some time ago, do we need to specify it at all? Maybe it already filters by the process.entity_id from the first event

Reply:
Ah was not aware of that, Samir had originally requested that we use original file name as well. For this context, I think OR is fine to alert if any MSBuild process or file was making network connections, so maybe another or here?

Reply:
Yeah, `or` is fine

Reply:
@w0rk3r thanks for the extra set of eyes. Updated according to our conversation.